tests/admin_views/tests.py

```diff
@@ -6069,3 +6069,39 @@ def test_implicitly_generated_pk(self):
         post_data = {'name': '2'}
         response = self.client.post(reverse('admin:admin_views_implicitlygeneratedpk_change', args=(1,)), post_data)
         self.assertEqual(response.status_code, 302)
+
+
+@override_settings(ROOT_URLCONF='admin_views.urls')
+class AdminHistoryViewTest(TestCase):
```

Comment:
Test -> Tests (and I suggest a new `test_history_view.py` file since this file is quite large already.

Reply:
Shall I refactor this, or make a separate ticket for this?

Reply:
Adding new code in a good location is fine.